### DISCUSSION TITLE
Realm migration issue (One or more errors occurred. (A Ruleset object already exists with primary key property ShortName == '"osu"'))

### DISCUSSION THREAD

**Discussion by Feodor0090** [ppy/osu#16655] [Q&A] [answered]
Same issue as #16651.
I installed karaoke and some other rulesets those i sometimes play before migration, but it didn't help. Maybe, there is an empty beatmap from accidental editor opening.
[attach_me.zip](https://github.com/ppy/osu/files/7951728/attach_me.zip)

**Comment by frenzibyte** [comment, marked as answer]
Indeed you have an empty beatmap with a missing ruleset (sentakki! dev build), here's an [archive of the fixed `client.realm`](https://github.com/ppy/osu/files/7951921/client.realm.zip) that you can put in place of the one you currently have (with the empty beatmap removed).

Marking as answered for being a duplicate of #16651, which is to be fixed by #16652.
